api/services/AuthorizationPolicy/index.js

```diff
@@ -15,6 +22,14 @@ policy.include('query', (p) => {
   p.include('zoomAccounts', (cp) =>
     cp.register('read', ({ viewer }) => isAdmin(viewer)),
   );
+
+  p.include('event', (cp) =>
+    cp.register(
+      'read',
+      ({ viewer, entity: event, value }) =>
+        isOrganiser(event, viewer) || isMember(viewer) || isPublicEvent(value),
```

Comment:
good work

Reply:
though the entity here will be undefined so pass value to isOrganiser. No need to access entity here.

Reply:
oh yes, fine